joern-cli/frontends/php2cpg/src/test/scala/io/joern/php2cpg/querying/NamespaceTests.scala

```diff
@@ -1,8 +1,18 @@
 package io.joern.php2cpg.querying
 
 import io.joern.php2cpg.testfixtures.PhpCode2CpgFixture
+import io.shiftleft.semanticcpg.language._
 
 class NamespaceTests extends PhpCode2CpgFixture {
+  "namespaces should be able to contain statements as top-level AST children" in {
+    val cpg = code("""<?php
+        |namespace foo {
+        |  echo 0;
+        |}
+        |""".stripMargin)
 
-  // TODO
+    inside(cpg.namespaceBlock.name("foo").l) { case List(ns) =>
+      ns.astChildren.l.map(_.code) shouldBe List("echo 0")
```

Comment:
```suggestion
      ns.astChildren.code.l shouldBe List("echo 0")
```

not that it makes any difference at all, but that's the DSLy way to write this